### ISSUE TITLE
Pseudocode followup issue

### ISSUE THREAD

**Issue by cwgoes** [cosmos/ibc#104]
Try to actually resolve the standard dependency graph and run the Typescript compiler.

Then we have typechecked pseudocode!

Ref https://github.com/cosmos/ics/issues/102
Ref https://github.com/cosmos/ics/pull/103

**Comment by cwgoes** [comment]
Blocked on https://github.com/cosmos/ics/issues/126 at minimum, not worth doing until more solidified.

**Comment by cwgoes** [comment]
I spent awhile trying this, but I don't think it's worth it, it will make the pseudocode too verbose.

Better to focus on formal verification - https://github.com/cosmos/ics/issues/42.

**Comment by gjermundgaraba** [comment]
This issue was mistakenly reopened during some integration work, so closing again now.
